### ISSUE TITLE
I cant do anything

### ISSUE THREAD

**Issue by Nzlovedota** [ValveSoftware/Dota2-Gameplay#6303]
### Description

Cant buy item (with 0 gold) in pick and draft time, in game cant do anything, just right click to move, but cant see my hero!

### Screenshots

![20221231165754_1](https://user-images.githubusercontent.com/121715233/210132994-d9d3fe22-6103-4bc4-b2b1-c231bd45d737.jpg)
![20221231170524_1](https://user-images.githubusercontent.com/121715233/210132995-4704c465-820e-4839-85a3-9c494769957a.jpg)

**Comment by Nzlovedota** [comment]
please bring me back to the ranked game Valve, i don't deserve low priority

**Comment by mbuchelly** [comment]
Hello i face this exact problem, in last year i found exactly same problem.

Problem appears when u accept a match and for some reason dota disconnect after you accept the match, then you click on reconnect and from that moment on picking a hero problem happens you can select any hero and you appear that you have 0 gold from buying before entering the match.

And when the match starts it shows exactly as on those photos

Match ID: 7086745855 

as you can see from replay and from IO perspective you will find exactly that photo if you choose on replay see io perspective

**Comment by Jmknights93** [comment]
> Hello i face this exact problem, in last year i found exactly same problem.
> 
> Problem appears when u accept a match and for some reason dota disconnect after you accept the match, then you click on reconnect and from that moment on picking a hero problem happens you can select any hero and you appear that you have 0 gold from buying before entering the match.
> 
> And when the match starts it shows exactly as on those photos
> 
> Match ID: 7086745855
> 
> as you can see from replay and from IO perspective you will find exactly that photo if you choose on replay see io perspective

Yup, this is bullshit. Happens to me at least once or twice a month...

![20230411210428_1](https://user-images.githubusercontent.com/125426193/231142002-04f09eaf-6860-4e39-bce9-c4435e5674d6.jpg)

**Comment by IgorHoruzha** [comment]
@RutyMan 
What operation system do You use?
How  often does it happened?
Are there any scenarios that may lead to such consequences on purpose?
Could You provide Your PC configuration, please?

Could You check Your RAM using [memtest86+](https://en.wikipedia.org/wiki/Memtest86), please?
Could You stress test Your CPU using [CPU-Z](https://en.wikipedia.org/wiki/CPU-Z), please?
Could You stress test Your GPU using [GPU-Z](https://en.wikipedia.org/wiki/GPU-Z), please?
Could You check DirectX functionality, please?
1. Press Windows + R, type dxdiag in the box, and press Enter.
2. Switch to the Display tab on the window. 
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/29505510/c164efba-bcfe-4f1a-8cc9-9816391ba83e)

**Comment by chrislandeza** [comment]
Getting the same problem. I have good internet connection but I keep getting disconnected
![image](https://github.com/user-attachments/assets/2a55c5a6-674b-42ae-8b85-8ac545650573)

**Comment by IgorHoruzha** [comment]
> Getting the same problem. I have good internet connection but I keep getting disconnected ![image](https://private-user-images.githubusercontent.com/4858242/364813768-58740907-4b59-4ffb-8077-3a7a6d6a553f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NTExNjYsIm5iZiI6MTcyNTU1MDg2NiwicGF0aCI6Ii80ODU4MjQyLzM2NDgxMzc2OC01ODc0MDkwNy00YjU5LTRmZmItODA3Ny0zYTdhNmQ2YTU1M2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MDVUMTU0MTA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTA1YzNkODU4NzEyZTEyYzE5ODk1ZTNkODZmZDczYmY1NWVlNTY4YTE0MzVmZmU5NGUzN2Y4MmVkNmViNWYwMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.7Viwym3TlzuPDQB4MwqtvTHsiJVj2BsGSOhk106cDR0)
> 
> ![image](https://private-user-images.githubusercontent.com/4858242/364814267-0b4db052-00ef-473d-bff3-f732185b3cdb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NTExNjYsIm5iZiI6MTcyNTU1MDg2NiwicGF0aCI6Ii80ODU4MjQyLzM2NDgxNDI2Ny0wYjRkYjA1Mi0wMGVmLTQ3M2QtYmZmMy1mNzMyMTg1YjNjZGIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MDVUMTU0MTA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Nzk1YWNhZmFkYjE3ZDdjYjlmYWU4MjE5YWY1ZGU2MmQ3MzFlMjc2ZjcxN2RiYTk5NWI2OTIyMTI0NDQ1NmI4NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.-Nyf2ct-sITe0EfCS6MYvcHyImFa2zHrnRfdvJIie3c)
> 
> Then I got abandon LOL

Could You provide:
1) How often do You have the issue, please?
2) Could You record a video of triggering the issue intestinally, please?

**Comment by commentator24** [comment]
still happening #30504
